README.md

```diff
@@ -118,7 +147,29 @@ entities (no need to draw an ERD)...
 Your answer...
 
 ```
-Replace this with your answer
+ERD stands for "Entity Relationship Diagram". It is uses to model the relationships of objects to one another.
+
+Genie:
```

Comment:
We were looking for an English description here. How Genie and Lamp are bound by a one-to-one relationship, and Person has many Pets, an Pets belong to one Person.